pkg/mgrconfig/load.go

```diff
@@ -215,6 +218,22 @@ func (cfg *Config) CompleteKernelDirs() {
 	cfg.KernelBuildSrc = osutil.Abs(cfg.KernelBuildSrc)
 }
 
+func (cfg *Config) CompleteKernelSubsystem() {
+	if len(cfg.KernelSubsystem) == 0 {
+		cfg.KernelSubsystem = []cover.Subsystem{
+			{
+				Name:  "all",
+				Paths: []string{""},
+			},
+		}
+	} else {
+		cfg.KernelSubsystem = append(cfg.KernelSubsystem, cover.Subsystem{
```

Comment:
The append will work for an empty slice as well, so you can leave just this append and remove the rest of the function.